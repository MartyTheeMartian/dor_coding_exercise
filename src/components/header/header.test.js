import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from './header';

describe('Header Component', () => {
  it('Header Component Renders', () => {
    const component = shallow(<Header />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
