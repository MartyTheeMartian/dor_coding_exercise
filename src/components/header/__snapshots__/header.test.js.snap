// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Header Component Renders 1`] = `
<tr
  className="store-header-wrapper"
>
  <td>
    Store Visits
  </td>
</tr>
`;
